jumpcloud-ADMU/Powershell/Public/Start-Reversion.ps1

```diff
@@ -0,0 +1,391 @@
+Function Start-Reversion {
+    <#
+    .SYNOPSIS
+        Reverts a user migration by restoring original registry files for a specified Windows SID.
+
+    .DESCRIPTION
+        This function reverts a user migration by:
+        1. Looking up the account SID in the Windows registry ProfileList
+        2. Identifying the .ADMU profile path
+        3. Restoring original NTUSER.DAT and UsrClass.dat files from backups
+        4. Preserving migrated files with _migrated suffix for rollback purposes
+
+    .PARAMETER UserSID
+        The Windows Security Identifier (SID) of the user account to revert.
+
+    .PARAMETER TargetProfileImagePath
+        The actual profile path to revert. If not specified, will use the path from the registry.
+        This path will be validated to ensure it exists and is associated with the UserSID.
+
+    .PARAMETER DryRun
+        Shows what actions would be performed without actually executing them.
+
+    .PARAMETER Force
+        Bypasses confirmation prompts and forces the revert operation.
+
+    .EXAMPLE
+        Revert-Migration -UserSID "S-1-5-21-123456789-1234567890-123456789-1001"
```

Comment:
This example would only work if the profile image path was valid right?

Reply:
Works now 
![Uploading image.png…]()